FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Keep_Alive/DemoTasks/KeepAliveMQTTExample.c

```diff
@@ -139,9 +139,9 @@
 /**
  * @brief Keep alive time reported to the broker while establishing an MQTT connection.
  *
- * It is the responsibility of the Client to ensure that the interval between
- * Control Packets being sent does not exceed the this Keep Alive value. In the
- * absence of sending any other Control Packets, the Client MUST send a
+ * It is the responsibility of the client to ensure that the interval between
+ * control packets being sent does not exceed the this keep-alive value. In the
```

Comment:
```suggestion
 * control packets being sent does not exceed this keep-alive value. In the
```

Reply:
Hmmm I feel like this is nice here above the macro.